src/main/java/com/ostapiuk/core/client/ReportPortalAPIClient.java

```diff
@@ -0,0 +1,28 @@
+package com.ostapiuk.core.client;
+
+import com.ostapiuk.core.properties.ConfigProperties;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public class ReportPortalAPIClient {
+
+    public RequestSpecification initSpec() {
+        return new RequestSpecBuilder()
+                .setContentType(ContentType.JSON)
+                .setBaseUri(ConfigProperties.getBaseApiUrlProperty())
+                .addFilter(new ResponseLoggingFilter())
+                .addFilter(new RequestLoggingFilter())
+                .build();
+    }
+
+    public Response getReportPortalStatus() {
+        return given().spec(initSpec())
+                .when().get();
```

Comment:
It is better to make a status check directly at the method. RestAssured has this posibility.
 .then().statusCode(SC_OK)